### PR TITLE
fix(forms-web-app): add env appropriate GA_ID value to each page

### PR DESCRIPTION
**Pull request by frank-dodion**
## Ticket Number
AS-1439

## Description of change
Add env appropriate GA_ID to each page and use this value to initialise GA.

### CODE REVIEW

File: packages/forms-web-app/src/views/includes/head.njk [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <!--[if lte IE 8]><link href="/public/stylesheets/main-ie8.css" rel="stylesheet" type="text/css" /><![endif]-->
   <!--[if gt IE 8]><!--><link href="/public/stylesheets/main.css" media="all" rel="stylesheet" type="text/css" /><!--<![endif]-->
 
+  <p class="govuk-visually-hidden" id="gaId">{{ googleAnalyticsId }}</p>
```
[the remark this answers is not in the document]
Comment (a6software) [reply packages/forms-web-app/src/views/includes/head.njk:6]: An alternative might be to use `<input type="hidden"...`

[the remark this answers is not in the document]
Comment (frank-dodion) [reply packages/forms-web-app/src/views/includes/head.njk:6]: It is the GDS style. What would you suggest? We could use other elements.

[the remark this answers is not in the document]
Comment (a6software) [reply packages/forms-web-app/src/views/includes/head.njk:6]: https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/helpers/_visually-hidden.scss#L5 - definitely going to be an issue for screen readers. Hidden input is viable I think.

[the remark this answers is not in the document]
Comment (frank-dodion) [reply packages/forms-web-app/src/views/includes/head.njk:6]: I could use the hidden attribute that works for modern browsers and that will hide it from screen readers too.

[the remark this answers is not in the document]
Comment (frank-dodion) [reply packages/forms-web-app/src/views/includes/head.njk]: I will update to use hidden attribute.

[the remark this answers is not in the document]
Comment (frank-dodion) [reply packages/forms-web-app/src/views/includes/head.njk]: @a6software Good point. Thanks.

[the remark this answers is not in the document]
Comment (a6software) [reply packages/forms-web-app/src/views/includes/head.njk]: https://stackoverflow.com/a/31107263 seems the aria-hidden attribute does something similar. 

I had no idea about `hidden`. 

:+1:

**Pull request merged**